### PR TITLE
Changed wording

### PR DESCRIPTION
**Pull request by get-itips**
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/413

### CODE REVIEW

**Comment by dariomws306** [comment]
Looks good.

**Comment by e0i** [comment]
cc @yogkumgit

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@tseward

Additional review needed for this PR. Thanks.

File: docset/windows/defender/defender.md [hunk 1]
```diff
@@ -22,8 +22,10 @@ ms.assetid: 00797342-0AAC-4FFF-84FE-4749DA9242D6
 
 # Defender Module
 ## Description
-This reference provides cmdlet descriptions and syntax for all Defender-specific cmdlets. 
-It lists the cmdlets in alphabetical order based on the verb at the beginning of the cmdlet.
+This reference provides functions descriptions and syntax for all Defender-specific functions. 
+It lists the functions in alphabetical order based on the verb at the beginning of the functions.
+
+[!NOTE] You might also hear these functions referred to as cmdlets. These functions were designed to feel like cmdlets even though they were developed as PowerShell functions.
```
Comment (JohanFreelancer9) [inline docset/windows/defender/defender.md]:
```suggestion
> [!NOTE]
> You might also hear these functions being referred to as cmdlets. They were designed to feel like cmdlets even though they were developed as PowerShell functions.
```

**Comment by tseward** [comment]
Looks good once the copy review is completed.

**Comment by rmca14** [comment]
No staging link was generated during validation.

**Pull request merged**